### ISSUE TITLE
Use Cypress to test application

### ISSUE THREAD

**Issue by j-000** [j-000/scratchpad#29]
I came across Cypress at work and it seems appropriate to use it to test this project considering there is no back-end to test.

With Cypress tests are written in JavaScript which makes things easier. 👍 

📚 [Cypress Docs](https://docs.cypress.io/guides/overview/why-cypress.html#In-a-nutshell) 

This would be a great opportunity to learn about this technology as well.

Any suggestions? 
@nimitbhargava @rasliche

**Comment by nimitbhargava** [comment]
@j-000 Unfortunately, I have never tried any front end testing tool before.
But would like to give it a try, it looks interesting! 🔥

**Comment by j-000** [comment]
@nimitbhargava  I would be learning as well. I have never used any front-end testing tool 😄 
I am currently giving myself about one week to just play around with the framework so I get used to the basics. I will look into adding it to the master branch by this weekend - unless someone is faster than me 😆

**Comment by crispinkoech** [comment]
Any update on this? I'd be interested in how you guys would write tests.

**Comment by j-000** [comment]
Currently learning the basics and also working on V2 (checkout the branch! :muscle:)
Have you got any xp with Cypress?

**Comment by crispinkoech** [comment]
Nah, I've only ever tested server-side code with NodeJS/Mocha. Lemme check out that branch :)

**Comment by j-000** [comment]
Same! :smile: 
Cypress is really good, it's mostly automated tests but there's a bit of a learning curve. :laughing:  I have only managed some basic stuff. 

Cool, let me know what you think.

**Comment by crispinkoech** [comment]
Dude, looks nice!!!

**Comment by j-000** [comment]
:blush: :laughing: Thanks!
